src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -43,4 +43,16 @@ public void setData(Model model) {
     protected void indicateAttemptToExecuteIncorrectCommand() {
         EventsCenter.getInstance().post(new IncorrectCommandAttemptedEvent(this));
     }
+
+    /**
+     * Does this command modify data in TaskBook? true or false
+     * @return
+     */
+
+
+    /***
+     * To return the command word child classes of Command will have
+     * @return
+     */
+    public abstract String getCommandWord();
```

Comment:
this.toString() returns seedu.address.logic.commands.AddFloatingTaskCommand@11fc6782